main.star

```diff
@@ -173,3 +185,52 @@ def get_l1_config(all_l1_participants, l1_network_params, l1_network_id):
     env_vars["L1_CHAIN_ID"] = str(l1_network_id)
     env_vars["L1_BLOCK_TIME"] = str(l1_network_params.seconds_per_slot)
     return env_vars
+
+
+def _install_faucet(
+    plan,
+    optimism_args,
+    l1_config_env_vars,
+    l1_priv_key,
+    deployment_output,
+    l2s,
+):
+    faucet_params = optimism_args.faucet
+    faucets = [
+        faucet.faucet_data(
+            name="l1",
+            chain_id=l1_config_env_vars["L1_CHAIN_ID"],
+            el_rpc=l1_config_env_vars["L1_RPC_URL"],
+            private_key=l1_priv_key,
+        ),
+    ]
+    for l2_num, l2 in enumerate(l2s):
+        chain = optimism_args.chains[l2_num]
+        chain_id = chain.network_params.network_id
```

Comment:
After #220, the `l2` has a `network_id` attribute so you no longer need to pass the `chains` just to get the chain ID (and you can drop the `optimism_args` altogether)